docs/tutorial/security.md

```diff
@@ -709,6 +709,97 @@ const { shell } = require('electron')
 shell.openExternal('https://example.com/index.html')
 ```
 
+## 15) Disable the `remote` module
+
+The `remote` module provides a simple way to do inter-process communication (IPC) between the renderer process (web page) and the main process. Using it, a renderer can invoke methods of a main process object without explicitly sending inter-process messages. If your desktop application does not run untrusted content, this is a very elegant way to have your renderer processes access and work with modules that are only available to the main process, such as GUI-related modules (dialog, menu etc.).
```

Comment:
provides a simple way for the renderer processes to access APIs normally only available in the main process

Reply:
I would also omit web page, it can just be the preload script calling those APIs

Reply:
Copied this from the `remote` documentation 😁 . But I agree with your changes, amended.